apis_configs/fence_settings.py

```diff
@@ -68,6 +68,17 @@
         ENABLED_IDENTITY_PROVIDERS = data['ENABLED_IDENTITY_PROVIDERS']
         APP_NAME = data['APP_NAME']
         HTTP_PROXY = data['HTTP_PROXY']
+        os.environ["GOOGLE_API_KEY"] = data['GOOGLE_API_KEY']
```

Comment:
Should this be a k8s secret as well?

Reply:
`fence_credentials.json` is already a secret, so this should be good ...